chef/cookbooks/network/recipes/default.rb

```diff
@@ -24,6 +24,27 @@
   p.run_action :install
 end
 
+if node[:network][:needs_openvswitch]
+  node[:network][:ovs_pkgs].each do |pkg|
+    p = package pkg do
+      action :nothing
+    end
+    p.run_action :install
+  end
+
+  unless ::File.exists?("/sys/module/#{node[:network][:ovs_module]}")
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.